.gitignore

```diff
@@ -373,7 +373,7 @@ BuildOutput/
 .nuget/
 
 # Installer build configuration header
-installer/dev/project_reunion_definitions_override.h
+installer/dev/windows_app_sdk_definitions_override.h
```

Comment:
Should be "windowsappsdk_definitions_override.h" per the name in the installer project file and in the definitions header.